yeat/cli.py

```diff
@@ -8,35 +8,43 @@
 # -------------------------------------------------------------------------------------------------
 
 import argparse
+from argparse import Action
+import json
 from pathlib import Path
 from pkg_resources import resource_filename
 from snakemake import snakemake
+import sys
 import yeat
+from yeat.config import AssemblerConfig
 
 
-ASSEMBLY_ALGORITHMS = ["spades", "megahit", "unicycler"]
+CONFIG_TEMPLATE = [
+    dict(
+        assembler="spades",
+        extra_args="",
+    ),
+    dict(
+        assembler="megahit",
+        extra_args="",
+    ),
+]
 
 
-def check_assemblers(assemblers):
-    algorithms = []
-    for algorithm in assemblers:
-        if algorithm not in ASSEMBLY_ALGORITHMS:
-            message = (
-                f"Found unsupported assembly algorithm with `--assemblers` flag: [[{algorithm}]]!"
-            )
-            raise ValueError(message)
-        if algorithm in algorithms:
-            message = (
-                f"Found duplicate assembly algorithm with `--assemblers` flag: [[{algorithm}]]!"
-            )
-            raise ValueError(message)
-        algorithms.append(algorithm)
+class InitAction(Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        json.dump(CONFIG_TEMPLATE, sys.stdout, indent=4)
+        print()
```

Comment:
Added this for my own OCD sanity. Without this, the terminal would display the template string without a newline character for the terminal command.